package.json

```diff
@@ -134,9 +134,6 @@
     "sort-package-json": "^2.8.0",
     "typescript": "~5.2.0"
   },
-  "jest": {
```

Comment:
Think we wanna keep this, and we should be using `msw` to mock out network requests in tests.